queue_timings.py

```diff
@@ -5,20 +5,21 @@
 
 import os.path
 # noinspection PyPep8Naming
-import pickle
 import math
 
 
 def main():
-    if os.path.isfile("bodyfetcherQueueTimings.p"):
```

Comment:
Maybe we still want to support both formats? If both files are present we can just merge the data.